src/BenchmarkDotNet/Toolchains/NativeAot/Generator.cs

```diff
@@ -203,6 +204,23 @@ private string GetTrimmingSettings()
             return sb.ToString();
         }
 
+        public IEnumerable<string> GetRdXmlFiles(Type benchmarkTarget, ILogger logger)
+        {
+            var projectFile = GetProjectFilePath(benchmarkTarget, logger);
+            var projectFileFolder = projectFile.DirectoryName;
+            yield return GeneratedRdXmlFileName;
+            var rdXml = Path.Combine(projectFileFolder, "rd.xml");
+            if (File.Exists(rdXml))
+            {
+                yield return rdXml;
+            }
+
+            foreach (var item in Directory.GetFiles(projectFileFolder, "*.rd.xml"))
+            {
+                yield return item;
+            }
```

Comment:
it seems like this could be simplified if we search for `*rd.xm` files:

```suggestion
            yield return GeneratedRdXmlFileName;
            
            var projectFileFolder = GetProjectFilePath(benchmarkTarget, logger).DirectoryName;
            foreach (var rdFile in Directory.GetFiles(projectFileFolder, "*rd.xml"))
            {
                yield return rdFile;
            }
```

Reply:
What if somebody have library which ends with `rd`? or just XML files. For example `Thunderbird.xml`, that would be captured.

Reply:
I am afraid that neither `*.rd.xml`  and `*rd.xml` are good filters, as users can call their files without rd prefix like `nativeaot_rules.xml`. But since it's a project with BenchmarkDotNet benchmarks we could just tell them that all that we support is `rd.xml`. @kant2002 what do you think?

@MichalStrehovsky what is your opinion on that? Are NativeAOT using names other than `rd.xml` for their `RdXmlFile` files?

Reply:
Based on my experience projects in the wild was using either just `rd.xml` or I recentrly start push for separate files per [library](https://github.com/kant2002/rdxmllibrary), to at least simplify mundane work. Naming convention which I propose is exaclly `nuget_package.rd.xml`. Also seems to be .NET Native Rd.xml also has approximately same convention, so that's why I choose that pattern.